gammagl/layers/conv/mgnni_m_iter.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+import tensorlayerx as tlx
+import tensorlayerx.nn as nn
+
+from gammagl.utils import degree
+from gammagl.layers.conv import MessagePassing
+
+
+class MGNNI_m_iter(MessagePassing):
```

Comment:
Remember to give a docstring refer to ``gcnconv``

Reply:
Thank you for the feedback. I apologize for the oversight. I have added the relevant docstring.